backend/app/routes/url/shorturl.js

```diff
@@ -0,0 +1,41 @@
+const validurl = require('valid-url');
+const shortid = require('shortid');
+const config = require('../../../config');
+const Url = require('../../models/Url');
+module.exports = async (req, res) => {
+  const { longUrl } = req.body;
+  const baseUrl = config.baseUrl;
+  // Check base url
+  if (!validurl.isUri(baseUrl)) {
+    return res.status(401).json('Invalid base url');
+  }
+  //Creating url code
+  const urlCode = shortid.generate();
+  //Check long url
+  if (validurl.isUri(longUrl)) {
```

Comment:
you should check this using validation schema
Please refer to other routes for ref